SU-HSDO/suhumsci
# STN-815: Fix placement/accessibility of contextual links on images on views

# READY FOR REVIEW

## Summary
A site manager should be able to access links in the contextual menu. The contextual menu should be correctly placed (in the upper right hand corner), regardless of image style.

## Need Review By (Date)
7/23

## Urgency
medium

## Steps to Test
1. In you CLI, run `npm run test` and confirm our test continue to pass.
2. Before reviewing the work in this PR I recommend switching to the sprint branch and reviewing the contextual links on images in the view(s) on these pages:
    * http://sparkbox-sandbox.suhumsci.loc/publications/book-shelf (issue exists at smaller screen sizes)
    * http://sparkbox-sandbox.suhumsci.loc/people/people-grid
    * http://sparkbox-sandbox.suhumsci.loc/backstop-testing/basic-card-views
3. You can also review [STN-815](https://sparkbox.atlassian.net/browse/STN-815) for a list of production sites where contextual links on images in views are broken (not fully accessible).
4. Checkout the `STN-815--contextual-menu-placement` branch and review the same pages listed above. 
    - [x] Confirm contextual links on images in views are accessible.
    - [x] Confirm all of the other contextual links remain unchanged, have appropriate spacing when there are multiple contextual links beside one another, and are still accessible.
